pkg/driver/efs_watch_dog.go

```diff
@@ -71,7 +71,7 @@ dns_name_suffix = sc2s.sgov.gov
 [mount-watchdog]
 enabled = true
 poll_interval_sec = 1
-unmount_grace_period_sec = 30
+unmount_grace_period_sec = 5
```

Comment:
What are the implications of changing this? I'm OK with changing it for a test, but it seems wrong to change it for everyone.